georocket-server/src/main/java/io/georocket/index/xml/BoundingBoxIndexer.java

```diff
@@ -305,17 +305,11 @@ private boolean validate() {
   
   @Override
   public Map<String, Object> getResult() {
-    if (!boundingBoxInitialized) {
```

Comment:
I'm not sure if it is necessary to split this method. Wouldn't it be possible to just override `getResult`, call `super.getResult()` and then customise the result? At least, `checkResult` shouldn't log or change `boundingBoxInitialized`. These are side effects and the name `checkResult` implies it's free of side effects. In addition, `boundingBoxInitialized` really only makes sense in `getResult`.